### ISSUE TITLE
yarle fails if "conversion.log" does not exist

### ISSUE THREAD

**Issue by luckman212** [akosbalasko/yarle#241]
### today I tried to use yarle@latest and it failed:

```
logfilepath: /Users/luke/Library/Application Support/yarle-evernote-to-md/conversion.log
node:fs:1263
  handleErrorFromBinding(ctx);
  ^

Error: ENOENT: no such file or directory, unlink '/Users/luke/Library/Application Support/yarle-evernote-to-md/conversion.log'
    at Object.unlinkSync (node:fs:1263:3)
    at Object.clearLogFile (/Users/luke/.npm/_npx/106a035a6ccf201d/node_modules/yarle-evernote-to-md/dist/utils/clearLogFile.js:26:8)
    at Object.run (/Users/luke/.npm/_npx/106a035a6ccf201d/node_modules/yarle-evernote-to-md/dist/dropTheRopeRunner.js:31:20)
    at Object.<anonymous> (/Users/luke/.npm/_npx/106a035a6ccf201d/node_modules/yarle-evernote-to-md/dist/dropTheRope.js:26:19)
    at Module._compile (node:internal/modules/cjs/loader:1092:14)
    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1121:10)
    at Module.load (node:internal/modules/cjs/loader:972:32)
    at Function.Module._load (node:internal/modules/cjs/loader:813:14)
    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:76:12)
    at node:internal/main/run_main_module:17:47 {
  errno: -2,
  syscall: 'unlink',
  code: 'ENOENT',
  path: '/Users/luke/Library/Application Support/yarle-evernote-to-md/conversion.log'
}
```

I created the logfile by executing `touch '/Users/luke/Library/Application Support/yarle-evernote-to-md/conversion.log'` and was then able to run successfully.

### not sure if this could be the problem?

extra leading slash (`/`) in `path.join` :
https://github.com/akosbalasko/yarle/blob/19a765f13570b075f44d5a9d2727816d3f2514e7/src/utils/logger.ts#L13

**Comment by akosbalasko** [comment]
not exactly, but it has already been reported here: https://github.com/akosbalasko/yarle/issues/239
so its a bug, I'll fix it soon.

**Comment by akosbalasko** [comment]
Fixed in version 4.0.4.
